### PR TITLE
deleted var

### PR DESCRIPTION
**Pull request by Abhiram-Gullapalli**
#### Related issues

Part of issue #16662 

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Abhiram-Gullapalli** [comment]
I'll be adding more commits to this pull request. Please do not merge it until then @teoli2003.

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by teoli2003** [comment]
> I'll be adding more commits to this pull request. Please do not merge it until then @teoli2003.
In these cases, mark it as "draft pull request" when you create it.
Capture d’écran 2022-07-06 à 16 05 33
That way, no risk of an error.

Please re-request review when this one is ready.

**Comment by Abhiram-Gullapalli** [comment]
sure

File: files/en-us/web/api/serviceworkerglobalscope/notificationclick_event/index.md [hunk 1]
```diff
@@ -56,8 +56,8 @@ self.addEventListener('notificationclick', function(event) {
   event.waitUntil(clients.matchAll({
     type: "window"
   }).then(function(clientList) {
-    for (var i = 0; i < clientList.length; i++) {
-      var client = clientList[i];
+    for (let i = 0; i < clientList.length; i++) {
+      const client = clientList[i];
```
Comment (teoli2003) [inline files/en-us/web/api/serviceworkerglobalscope/notificationclick_event/index.md:60]: That was correct, but this is a use case for `for...of`:
```suggestion
    for (const client of clientList) {
```

File: files/en-us/web/api/serviceworkerglobalscope/notificationclick_event/index.md [hunk 2]
```diff
@@ -79,8 +79,8 @@ self.onnotificationclick = function(event) {
...
```
Comment (teoli2003) [inline files/en-us/web/api/serviceworkerglobalscope/notificationclick_event/index.md:83]:
```suggestion
    for (const client of clientList) {
```

File: files/en-us/web/api/sharedworkerglobalscope/connect_event/index.md [hunk 3]
```diff
@@ -56,10 +56,10 @@ The connecting port can be referenced through the event object's `ports` paramet
 
 ```js
 self.onconnect = function(e) {
-    var port = e.ports[0];
+    const port = e.ports[0];
 
     port.onmessage = function(e) {
-      var workerResult = 'Result: ' + (e.data[0] * e.data[1]);
+      cosnt workerResult = 'Result: ' + (e.data[0] * e.data[1]);
```
Comment (teoli2003) [inline files/en-us/web/api/sharedworkerglobalscope/connect_event/index.md:62]:
```suggestion
      const workerResult = 'Result: ' + (e.data[0] * e.data[1]);
```

File: files/en-us/web/api/sourcebufferlist/index.md [hunk 4]
```diff
@@ -46,11 +46,11 @@ This example shows how to access the active source buffers of the {{domxref("Med
...
-var video = document.querySelector('video');
-var mediaSource = video.srcObject;
-var sourceBufferList = mediaSource.activeSourceBuffers;
-for (var i = 0; i < sourceBufferList.length; i++) {
-  var sourceBuffer = sourceBufferList[i];
+const video = document.querySelector('video');
+const mediaSource = video.srcObject;
+const sourceBufferList = mediaSource.activeSourceBuffers;
+for (let i = 0; i < sourceBufferList.length; i++) {
+  const sourceBuffer = sourceBufferList[i];
```
Comment (teoli2003) [inline files/en-us/web/api/sourcebufferlist/index.md:53]:
```suggestion
for (const sourceBuffer of sourceBufferList) {
```

File: files/en-us/web/api/speechsynthesis/index.md [hunk 5]
```diff
@@ -66,24 +66,23 @@ Now we'll look at a more fully-fledged example. In our [Speech synthesizer demo]
...
-var synth = window.speechSynthesis;
+const synth = window.speechSynthesis;
 
-var inputForm = document.querySelector('form');
-var inputTxt = document.querySelector('.txt');
-var voiceSelect = document.querySelector('select');
+const inputForm = document.querySelector('form');
+const inputTxt = document.querySelector('.txt');
+cosnt voiceSelect = document.querySelector('select');
+cosnt pitch = document.querySelector('#pitch');
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesis/index.md:74]:
```suggestion
const voiceSelect = document.querySelector('select');
const pitch = document.querySelector('#pitch');
```

File: files/en-us/web/api/speechsynthesis/index.md [hunk 6]
```diff
@@ -66,24 +66,23 @@ Now we'll look at a more fully-fledged example. In our [Speech synthesizer demo]
...
+const rate = document.querySelector('#rate');
+const rateValue = document.querySelector('.rate-value');
 
-var pitch = document.querySelector('#pitch');
-var pitchValue = document.querySelector('.pitch-value');
-var rate = document.querySelector('#rate');
-var rateValue = document.querySelector('.rate-value');
-
-var voices = [];
+let voices = [];
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesis/index.md:79]:
```suggestion
const voices = [];
```

File: files/en-us/web/api/speechsynthesis/voiceschanged_event/index.md [hunk 7]
```diff
@@ -31,12 +31,12 @@ A generic {{DOMxRef("Event")}} with no added properties.
 This could be used to repopulate a list of voices that the user can choose between when the event fires. You can use the `voiceschanged` event in an [`addEventListener`](/en-US/docs/Web/API/EventTarget/addEventListener) method:
 
 ```js
-var synth = window.speechSynthesis;
+const synth = window.speechSynthesis;
 
 synth.addEventListener('voiceschanged', function() {
-  var voices = synth.getVoices();
+  const voices = synth.getVoices();
   for(i = 0; i < voices.length ; i++) {
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesis/voiceschanged_event/index.md:38]:
```suggestion
  for(let i = 0; i < voices.length ; i++) {
```

File: files/en-us/web/api/speechsynthesis/voiceschanged_event/index.md [hunk 8]
```diff
@@ -48,11 +48,11 @@ synth.addEventListener('voiceschanged', function() {
 Or use the `onvoiceschanged` event handler property:
 
 ```js
-var synth = window.speechSynthesis;
+const synth = window.speechSynthesis;
 synth.onvoiceschanged = function() {
-  var voices = synth.getVoices();
+  const voices = synth.getVoices();
   for(i = 0; i < voices.length ; i++) {
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesis/voiceschanged_event/index.md:54]:
```suggestion
  for(let i = 0; i < voices.length ; i++) {
```

File: files/en-us/web/api/speechsynthesiserrorevent/error/index.md [hunk 9]
```diff
@@ -62,21 +62,21 @@ A string containing an error code. Possible codes are:
...
   ...
 
 inputForm.onsubmit = function(event) {
   event.preventDefault();
 
-  var utterThis = new SpeechSynthesisUtterance(inputTxt.value);
-  var selectedOption = voiceSelect.selectedOptions[0].getAttribute('data-name');
+  const utterThis = new SpeechSynthesisUtterance(inputTxt.value);
+  const selectedOption = voiceSelect.selectedOptions[0].getAttribute('data-name');
   for(i = 0; i < voices.length ; i++) {
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesiserrorevent/error/index.md:80]:
```suggestion
  for(let i = 0; i < voices.length ; i++) {
```

File: files/en-us/web/api/speechsynthesiserrorevent/index.md [hunk 10]
```diff
@@ -33,21 +33,21 @@ _`SpeechSynthesisErrorEvent` extends the {{domxref("SpeechSynthesisEvent")}} int
...
   ...
 
 inputForm.onsubmit = function(event) {
   event.preventDefault();
 
-  var utterThis = new SpeechSynthesisUtterance(inputTxt.value);
-  var selectedOption = voiceSelect.selectedOptions[0].getAttribute('data-name');
+  const utterThis = new SpeechSynthesisUtterance(inputTxt.value);
+  const selectedOption = voiceSelect.selectedOptions[0].getAttribute('data-name');
   for(i = 0; i < voices.length ; i++) {
```
Comment (teoli2003) [inline files/en-us/web/api/speechsynthesiserrorevent/index.md:51]:
```suggestion
  for(let i = 0; i < voices.length ; i++) {
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**